test/extensions/filters/udp/dns_filter/dns_filter_integration_test.cc

```diff
@@ -97,6 +106,42 @@ INSTANTIATE_TEST_SUITE_P(IpVersions, DnsFilterIntegrationTest,
                          testing::ValuesIn(TestEnvironment::getIpVersionsForTest()),
                          TestUtility::ipTestParamsToString);
 
+TEST_P(DnsFilterIntegrationTest, DISABLED_ExternalLookupTest) {
```

Comment:
Why is this disabled?

Reply:
In CI, It appears that we cannot actually do a resolution to an external name server.   These tests pass locally but fail in CI.  Since there is an equivalent unit test, and coverage is acceptable, I can remove these from the integration test, unless you have objections, and/or I'm missing something about how the CI works.

Reply:
CI needs to be fully jailed and self contained. Optimally you would set up a fake upstream resolver and respond to queries that way. See the UDP proxy integration tests for how to do this.